scene/gui/text_edit.h

```diff
@@ -290,93 +263,93 @@ class TextEdit : public Control {
 	Array st_args;
 	bool draw_control_chars = false;
 
-	uint32_t version;
-	uint32_t saved_version;
+	uint32_t version = 0;
+	uint32_t saved_version = 0;
 
-	int max_chars;
-	bool readonly;
-	bool indent_using_spaces;
-	int indent_size;
-	String space_indent;
+	int max_chars = 0;
+	bool readonly = true; // Initialise to opposite first, so we get past the early-out in set_readonly.
```

Comment:
This is set_readonly function
```
void TextEdit::set_readonly(bool p_readonly) {
	if (readonly == p_readonly) {
		return;
	}

	readonly = p_readonly;
	_generate_context_menu();

	update();
}
```
It reads current value of `readonly` variable, so if there was no `readonly = false` before, the behavior of this function would be unpredictable the first time, because the variable `readonly` would be uninitialized and could have `false` or `true` value inside.

I think that enough is to remove call to this function, change `readonly` variable to false and put at the end of constructor:
```
	_generate_context_menu();
	update();
```

Reply:
Makes sense yeah.